quickstarts/microsoft.web/app-service-managed-certificate-apex/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "SubjectName": {
```

Comment:
use the GET-PREREQ tokens for the param values here

Reply:
You need to match the GET-PREREQ tokens for all parameters with the outputs of the prereq templates.  Look at other examples in the repo if you need a guide (or the contribution guide itself).